styles/login-signup.css

```diff
@@ -0,0 +1,25 @@
+.form {
+  font-size: 1rem;
+  box-shadow: 1px 1px 1px 1px var(--shadow-color);
+}
+
+.link-redirect {
+  text-decoration: underline;
+  color: var(--primary-color);
+  cursor: pointer;
+  font-size: 1.2rem;
+}
+.input-text {
+  width: 18rem;
+  margin-left: 2rem;
+}
+
```

Comment:
add input-text:invalid class for error handling from CSS it will be a better user experience